docs/resources/android.svg

```diff
@@ -1,17 +1 @@
- </g>
-</svg>
+<svg height="100" viewBox="0 0 26.458333 26.458334" width="100" xmlns="http://www.w3.org/2000/svg"><g transform="translate(0 -270.54)"><path d="m0 270.54h26.458v26.458h-26.458z" fill="#a4c639" stroke-width=".27763"/><path d="m21.059 281.63c0-.62897-.50779-1.1406-1.1391-1.1406-.63126 0-1.1421.50935-1.1421 1.1422v4.7769c0 .62508.51165 1.1421 1.1445 1.1421.62818 0 1.1391-.51705 1.1391-1.1421v-4.7846h-.0079zm-12.929-.92221.0032 7.4085c0 .68298.54328 1.227 1.2139 1.227h.82033v2.5312c0 .63281.50933 1.1437 1.1344 1.1437.62509 0 1.1321-.50932 1.1421-1.1328v-2.5312h1.538v2.5312c0 .63511.50933 1.1444 1.1344 1.1444.63512 0 1.1437-.50932 1.1437-1.1499v-2.5312h.83191c.67139 0 1.2139-.54791 1.2139-1.2178v-7.4316l-10.209.006.03086.002zm7.4455-2.2611c-.23923 0-.43216-.1928-.43216-.43214 0-.23538.19294-.43063.43216-.43217.23923 0 .43216.19688.43216.43217 0 .23917-.19294.43214-.43216.43214m-4.6766 0c-.23923 0-.43216-.1928-.43216-.43214 0-.23684.19294-.43063.43216-.43063.23923 0 .43216.19688.43216.43987 0 .24297-.19449.43757-.43988.43757m4.8541-2.238.7941-1.4593c.04629-.0811.01079-.17442-.05866-.21992-.08027-.0347-.17287-.0117-.21916.0694l-.8103 1.4662c-.66985-.31253-1.4323-.48616-2.2303-.48616-.79795 0-1.5573.16596-2.2318.46534l-.79795-1.4686c-.04089-.0755-.13892-.10413-.21452-.0618-.07718.0347-.10494.13884-.06175.20826l.79487 1.447c-1.5666.80799-2.6238 2.346-2.6238 4.1132h10.285c0-1.7672-1.0557-3.3053-2.6207-4.1132m-9.2127 4.3002c-.63049 0-1.1421.51319-1.1421 1.1445v4.7785c0 .63281.51242 1.1421 1.1452 1.1421.62817 0 1.1398-.50932 1.1398-1.1421v-4.7769c0-.62897-.50933-1.1406-1.1344-1.1406" fill="#fff" stroke-width=".77172"/></g></svg>
```

Comment:
I'm not sure about changes like this and what is actually being done as this is a vector image file.

Reply:
ImageOptim used SVGO under the hood to optimize SVGs. It optimizes all paths and removes some whitespace. It does not change the overall shape

![image](https://user-images.githubusercontent.com/5244986/57579980-6f2a1c00-74a4-11e9-9473-f96816c973b4.png)

See https://github.com/xbmc/xbmc/blob/ee8fd823e7afdf3bb3cd0d93c7c41cf87983cf16/docs/resources/android.svg